digi/xbee/io.py

```diff
@@ -621,8 +621,7 @@ def get_analog_value(self, io_line):
 
 class IOMode(Enum):
     """
-    Enumerates the different Input/Output modes that an IO line can be 
-    configured with.
+    Enumerates the different Input/Output modes that an IO line can be configured with.
```

Comment:
NP: It seems this line is over 80 characters. The PEP8 linter will show a warning, I guess.

Reply:
Done.